docs/_layouts/default.html

```diff
@@ -12,9 +12,9 @@
 
     {% include nav-home.html %}
 
-    <div id="content">
+    <main id="content" role="main">
```

Comment:
Just a question about that but it seems `main` tag isn't supported by IE10 (https://caniuse.com/#feat=html5semantic). 
What's the lower release of IE we support in our documentation ?